README.md

```diff
@@ -1,2 +1,28 @@
 # Hiking-Hub
-A Hub for finding details about Hiking spots near you
+
+Description
+Hiking Hub is an application developed for hikers that live in any urban or suburban areas that want to find a hiking trail destination. The Hiking Hub will enable the user to enter their location and will provide the nearest hiking trail locations. Once the locations are populated, the user can decide on the trail they are most interested in. Once the trail button is clicked, reddit results for that trail will become visible for the user to review. 
+
+User Story
+As A new hiker moving to a major city
+I Want to have a way to find nearby hiking trails that are close to my location
+SO THAT I can get to trails that are accessible to me and convenient. 
+
+
+
+Technologies Used
+As A new hiker moving to a major city
```

Comment:
Materialize CSS and Tailwind CSS are used for styling
Three API endpoints are utilized - Google Map API to determine trails, Open Weather API to get coordinates and Reddit API for Reddit subreddits
Local storage is used to store previous city searches